app/api/groups/accept_leave_requests.robustness.feature

```diff
@@ -0,0 +1,97 @@
+      | 16 | 13              | 151            |
+    And the database has the following table 'group_pending_requests':
+      | group_id | member_id | type          |
+      | 13       | 21        | invitation    |
+      | 13       | 31        | leave_request |
+      | 13       | 141       | leave_request |
+      | 13       | 161       | join_request  |
+      | 14       | 11        | invitation    |
+      | 14       | 21        | join_request  |
+
+  Scenario: Fails when the user is not a manager of the parent group
+    Given I am the user with id "21"
+    When I send a POST request to "/groups/13/leave-requests/accept?group_ids=31,141,21,11,13"
+    Then the response code should be 403
+    And the response error message should contain "Insufficient access rights"
+    And the table "groups_groups" should stay unchanged
+    And the table "group_pending_requests" should stay unchanged
+    And the table "group_membership_changes" should be empty
+    And the table "groups_ancestors" should stay unchanged
+
+  Scenario: Fails when the user is a manager of the parent group, but doesn't have enough rights to manage memberships
+    Given I am the user with id "21"
+    And the database has the following table 'group_managers':
+      | group_id | manager_id | can_manage |
+      | 13       | 21         | none       |
+    When I send a POST request to "/groups/13/leave-requests/accept?group_ids=31,141,21,11,13"
+    Then the response code should be 403
+    And the response error message should contain "Insufficient access rights"
+    And the table "groups_groups" should stay unchanged
+    And the table "group_pending_requests" should stay unchanged
+    And the table "group_membership_changes" should be empty
+    And the table "groups_ancestors" should stay unchanged
+
+  Scenario: Fails when the parent group id is wrong
+    Given I am the user with id "21"
+    When I send a POST request to "/groups/abc/leave-requests/accept?group_ids=31,141,21,11,13"
+    Then the response code should be 400
+    And the response error message should contain "Wrong value for parent_group_id (should be int64)"
+    And the table "groups_groups" should stay unchanged
+    And the table "group_pending_requests" should stay unchanged
+    And the table "group_membership_changes" should be empty
+    And the table "groups_ancestors" should stay unchanged
+
+  Scenario: Fails when group_ids is wrong
+    Given I am the user with id "21"
+    When I send a POST request to "/groups/13/leave-requests/accept?group_ids=31,abc,11,13"
+    Then the response code should be 400
+    And the response error message should contain "Unable to parse one of the integers given as query args (value: 'abc', param: 'group_ids')"
+    And the table "groups_groups" should stay unchanged
+    And the table "group_pending_requests" should stay unchanged
+    And the table "group_membership_changes" should be empty
+    And the table "groups_ancestors" should stay unchanged
```

Comment:
`expires_at` is ignored by GroupGroupStore.Transition()

Reply:
I don't get how it can be fully ignored:
- creating invitation/join request: should not be possible if member, but should be if expired 
- same for accepting invitation/request
For leave request, it is not that important indeed.

Do I create a separate issue for that?
Pff this expiration is very annoying. (we could brainstorm to see if we find better ideas)

Reply:
fixed

Reply:
improved the scenario in accept_leave_requests.feature